bin/notificators/hard_threshold_notificator.py

```diff
@@ -45,11 +45,13 @@ def send_slack_notification(self, query, is_send_to_user=True, is_send_to_admin_
             user=query.executing_user
         )
 
-        text = self.config.SLACK_HARD_THRESHOLD_MESSAGE.format(**params)
+        text_1 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_1.format(**params)
+        text_2 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_2.format(**params)
+        text_3 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_3.format(**params)
         if is_send_to_admin_channel:
-            self.send_slack_to_channel(text)
+            self.send_slack_to_channel(text_1 + text_2)
```

Comment:
🟡 
Imagine you're using `athena_alerter`. 
Do you expect to handle appending `\n` yourself to entries?

---
Format the messages by introducing new lines, instead of requiring the user to add them in `settings.py`.

You can introduce a helper function
```
def _format_lines(lines: Iterable[str]) -> str
```
in `notificator.py` and use it in `send_slack_to_user` or here

Reply:
👎 I am not convinced if it is worth using additional functions to make something as simple as a newline. In our particular case, we need an additional message that starts with a newline, but in other cases it may be a continuation of the first message.

Reply:
I think you should still append at least a new line or a " " (space) between lines automatically.

It's as simple as:
```

@staticmethod
def _format_lines(lines: Iterable[str]):
    return "\n".join(lines)
```

You can skip declaring the new method if you feel it's better.

Reply:
Done 👍